src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -102,6 +105,21 @@ public void addMessage(Message message) {
     return messagesInConversation;
   }
 
+  /** Access the current set of Messages sent by a specific user. */
+  public Queue<Message> getMessagesByOwner(UUID authorid) {
```

Comment:
Please make authorid lower camelcase (see previously provided link). I think getMessagesByAuthor would be a better name for this method.